sklearn/linear_model/_base.py

```diff
@@ -232,7 +232,7 @@ def predict(self, X):
 
         Returns
         -------
-        C : array, shape (n_samples,)
+        C : array, shape (n_samples,) or (n_samples, n_outputs)
```

Comment:
There are some parent classes, such as `TheilSenRegressor`, that can fit on multiple outputs. This can lead to some confusion for a reader.

May we revert this?

Reply:
I don't really understand your comment.
Do you mean that for multiclass classifiers like `LogisticRegression`, the return shape of `predict` will be `(n_samples,)` while having `n_classes` different values, so the "or (n_samples, n_outputs)" part can be confusing?

As far as I know for classifiers the `predict` function is inherited from `LinearClassifierMixin`, so I guess they will not be affected by this change.

Reply:
Sorry I did mistyped out my previous comment, `TheilSenRegressor` does **not** `fit` on multiple outputs and would not return a `(n_samples, n_outputs)` array for `predict`.

`TheilSenRegressor` inherits from `LinearModel` so it would inherit the docstring for `predict`. This means that the "or (n_samples, n_outputs)" part can be confusing for users looking at the docstring for `TheilSenRegressor.predict`. (Although it is technically a true statement because of "or")